package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "ajax",
+  "version": "1.0.0",
+  "description": "A tiny XMLHTTPRequest abstraction",
+  "main": "Ajax.js",
```

Comment:
bower.json is not using using the minified version, should I change both to the minified one?